numbers/logDistribution.go

```diff
@@ -8,25 +8,29 @@ import (
 //This is ideal for very small probabilities to avoid underflow.
 func BinomialDistLog(n int, k int, p float64) float64 {
 	coefficient := BinomCoefficientLog(n, k)
-	s := LogPowInt(p, k)
-	f := LogPowInt(1.0-p, n-k)
-	expression := MultiplyLog(s, f)
+	expression := BinomialExpressionLog(n, k, p)
 	return MultiplyLog(coefficient, expression)
 }
 
+func BinomialExpressionLog(n int, k int, p float64) float64 {
+	s := LogPow(p, float64(k))
+	f := LogPow(1.0-p, float64(n-k))
+	return MultiplyLog(s, f)
+}
+
 //BinomialDistLogMap returns log(BinomialDist), where log is the natural logarithm.
 //This function is similar to BinomialDistLog but passes in a map[int][]float64, where the int key
 //refers to n and the []float64 map values are the corresponding binomial coefficients for index k.
 //Useful to not recalculate the binomial coefficient each time when binomial densities must be constantly evaluated in logSpace, like in MCMC.
-func BinomialDistLogSlice(n int, k int, p float64, binomMap [][]float64) float64 {
+func BinomialDistLogSlice(n int, k int, p float64, binomCache [][]float64) float64 {
 	//DEBUG: fmt.Printf("n: %v. k: %v. len(binomMap): %v.", n, k, len(binomMap))
-	if binomMap[n] == nil {
-		binomMap[n] = AddBinomMapEntry(n)
+	if binomCache[n] == nil {
+		binomCache[n] = AddBinomMapEntry(n)
 	}
-	s := LogPowInt(p, k)
-	f := LogPowInt(1.0-p, n-k)
+	s := LogPow(p, float64(k))
```

Comment:
I think some of these changes are already in main.  I think the thing to do is to change over to main, get the newest version by running git pull, and then going back to this branch and merging in main.